src/core/public/core_system.ts

```diff
@@ -109,14 +108,18 @@ export class CoreSystem {
     this.savedObjects = new SavedObjectsService();
     this.uiSettings = new UiSettingsService();
     this.overlay = new OverlayService();
-    this.chrome = new ChromeService({ browserSupportsCsp });
+
     this.docLinks = new DocLinksService();
     this.rendering = new RenderingService();
     this.application = new ApplicationService();
     this.integrations = new IntegrationsService();
     this.deprecations = new DeprecationsService();
 
     this.coreContext = { coreId: Symbol('core'), env: injectedMetadata.env };
+    this.chrome = new ChromeService({
+      browserSupportsCsp,
+      packageInfoVersion: injectedMetadata.version,
+    });
```

Comment:
NIT/optional: The constructor order could benefit from some reordering. I think we should move the `this.coreContext = {...}` line just after the `injectedMetadata` initialization to avoid moving services around if/when they'll need to access `env`

Reply:
[Good idea!](https://github.com/elastic/kibana/pull/102443/files#diff-8bb0fa43e19d24795e11f84500c3a2a9cff3275a5f89fe3c47cf2fb156c493c6R100)